packages/experiment-solidbench/lib/ExperimentSolidBench.ts

```diff
@@ -183,6 +167,16 @@ export class ExperimentSolidBench implements Experiment {
     const network = networkHandler.network.id;
     const endpointProcessHandler = await this.hookSparqlEndpoint.start(context, { docker: { network }});
 
+    // Setup query loader and result serializer
+    const queryLoader = new QueryLoaderFile({
+      path: Path.join(context.experimentPaths.generated, 'out-queries'),
+      extensions: [ '.sparql' ],
+    });
+    const resultSerializer = new ResultSerializerCsv({
```

Comment:
I guess `ResultSerializerCsv` can be instantiated without args, as those are the default options?